include/pyre3.14t/modsupport.h

```
@@ -178,16 +193,58 @@ static inline int _PyPyre_ArgConvert(PyObject *arg, const char **format,
         *va_arg(*va, PyObject **) = arg;
         return 1;
     }
+    case '(': {
+        /* A sequence whose items the units up to the matching `)` convert.
+           `str` is excluded the way `converttuple` excludes it: a two-item
+           string is not a two-item argument list. */
+        Py_ssize_t items = 0, ignored = 0;
+        _PyPyre_ArgCount(*format, &items, &ignored, NULL);
+        char buffer[128];
+        if (!PySequence_Check(arg) || PyUnicode_Check(arg)) {
```

Clang (14.0.6)

[note] +2, including nesting penalty of 1, nesting level increased to 2

(clang)